packages/owl-bot/src/copy-code.ts

```diff
@@ -580,7 +582,9 @@ command in a local clone of this repo:
       WithRegenerateCheckbox.Yes,
       apiList,
       Force.Yes,
-      logger
+      logger,
+      undefined,
+      draftPulledRequests
```

Comment:
Consider refactoring to an options hash with defaults to avoid needing to specify N optional fields positionally.

Reply:
that sounds appealing